app/views/land-values.njk

```diff
@@ -28,21 +27,33 @@
     }) }}
     {% endif %}
 
-    <h1 class="govuk-heading-l">{{title}}</h1>
 
-    <form method="POST" autocomplete="off" novalidate >
-      {% for standard in components.inputs %}
-        {{ govukInput(standard) }}
-      {% endfor %}
 
-      {{ govukDetails(components.details) }}
+    <form method="POST" autocomplete="off" novalidate >
+      {% if errorText %}
```

Comment:
This `if` block could be simplified to
`<div class="govuk-form-group {{ "govuk-form-group--error" if errorText }}">`

Reply:
Good spot. Fixed.